src/components/userPreferencesModal/UserPreferencesModal.styl

```diff
@@ -0,0 +1,152 @@
+@import './../../design/styles/common/navbar.styl'
+@import './../../design/styles/common/global.styl'
+
+.nav
+  padding-left: 0;
+  list-style: none;
+
+  > li
+    position: relative;
+    display: block;
+
+  > li
+    float: left;
+    margin-bottom: -1px;
+
+.modal-open
+  overflow: hidden;
+
+.modal
+  overflow: hidden;
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 1050;
+  -webkit-overflow-scrolling: touch;
+  outline: 0;
+
+  // When fading in the modal, animate it to slide down
+  &.fade .modal-dialog
+    -webkit-transition: transform .3s ease-out;
+    -o-transition: transform .3s ease-out;
+    transition: transform .3s ease-out;
+    -webkit-transform: translate(0,-25%);
+    -ms-transform: translate(0,-25%);
+    -o-transform: translate(0,-25%);
+    transform: translate(0,-25%);
+
+  &.in .modal-dialog
+    transform: translate(0,0)
+
+.modal-open .modal
+  overflow-x: hidden;
+  overflow-y: auto;
+
+.modal-dialog
+  position: relative;
+  width: auto;
+  margin: 10px;
+
+.modal-content
+  position: relative;
+  background-color: #fff;
+  border: 1px solid rgba(0,0,0,.2);
+  border-radius: 6px;
+  -webkit-box-shadow: 0 3px 9px rgba(0,0,0,.5);
+  box-shadow: 0 3px 9px rgba(0,0,0,.5);
+  background-clip: padding-box;
+  outline: 0;
+
+.modal-backdrop
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 1050;
+  background-color: #fff;
+
+  &.fade
+    opacity: 0;
+
+  &.in
+    opacity: 0;
+
+.modal-header
+  padding: 15px;
+  border-bottom: 1px solid #e5e5e5;
+
+.modal-header .close
+  margin-top: -2px;
+
+.modal-title
+  margin: 0;
+  line-height: 1.42857143;
+
+.modal-body
+  position: relative;
+  padding: 15px;
+
+.modal-footer
+  padding: 15px;
+  text-align: right;
+  border-top: 1px solid #e5e5e5;
+
+  .btn+.btn
+    margin-left: 5px;
+    margin-bottom: 0;
+
+  .btn-group .btn+.btn
+    margin-left: -1px;
+
+  .btn-block+.btn-block
+    margin-left: 0;
+
+.modal-scrollbar-measure
+  position: absolute;
+  top: -9999px;
+  width: 50px;
+  height: 50px;
+  overflow: scroll;
+
+@media (min-width: 768px)
+  .modal-dialog
+    width: 600px;
+    margin: 30px auto;
+
+  .modal-content
+    -webkit-box-shadow: 0 5px 15px rgba(0,0,0,.5);
+    box-shadow: 0 5px 15px rgba(0,0,0,.5);
+
+@media (min-width: 992px)
+  .modal-lg
+    width: 900px;
+
+button.close
+  -webkit-appearance: none;
+  padding: 0;
+  cursor: pointer;
+  background: 0 0;
+  border: 0;
+
+.close
+  float: right;
+  font-size: 21px;
+  font-weight: 700;
+  line-height: 1;
+  color: #000;
+  text-shadow: 0 1px 0 #fff;
+  opacity: .2;
+
+.row
+  margin-right: -15px;
+  margin-left: -15px;
+
+.pull-left
+  float: left!important;
+
+ol, ul
+  margin-top: 0;
```

Comment:
Again, unless these are scoped, things like this feel dangerous

Reply:
I will change to keep component scoped